### ISSUE TITLE
Allow excluding exception messages from ExceptionTelemetry

### ISSUE THREAD

**Issue by nickguletskii** [microsoft/ApplicationInsights-dotnet#627]
It would be nice to have the ability to exclude exception messages from the telemetry, as the messages could contain PII.

**Comment by SergeyKanzhelev** [comment]
Thanks for feedback! This feature is partially blocked on this one: https://github.com/Microsoft/ApplicationInsights-dotnet/issues/498 Once the exception model will be exposed - you'd be able to override all fields in telemetry processor.

We will schedule those issues together
